src/platforms/java/common/gradle.mdx

```diff
@@ -0,0 +1,78 @@
+---
+title: Gradle
+sidebar_order: 100
+description: "Learn about using the Sentry Gradle Plugin."
+---
+
+The [Sentry Gradle Plugin](https://github.com/getsentry/sentry-android-gradle-plugin) is an addition to the main Java SDK and offers
+seamless integration with the Gradle build system. It supports the report of external dependencies of the application as part of events.
+
+## Setup
+
+### Install
+
+Using Gradle in your application module's `build.gradle` add:
+
+```groovy
+buildscript {
+    repositories {
+        mavenCentral()
+    }
+}
+
+plugins {
+    id "io.sentry.android.gradle" version "{{ packages.version('sentry.java.android.gradle-plugin', '3.0.0') }}"
+}
+```
+
+```kotlin
+buildscript {
+    repositories {
+        mavenCentral()
+    }
+}
+
+plugins {
+    id("io.sentry.android.gradle") version "{{ packages.version('sentry.java.android.gradle-plugin', '3.0.0') }}"
+}
+```
+
+<Note>
+
+Although the plugin id contains `android`, it will not do anything for a non-android project, besides collecting external dependencies.
+
+</Note>
+
+### Configure
+
+We expose the following configuration values directly in `build.gradle`:
+
+```groovy
+sentry {
+    // Disables or enables the reporting of dependencies metadata for Sentry.
+    // If enabled the plugin will collect external dependencies and will take care of
+    // uploading them to Sentry as part of events. If disabled, all the logic
+    // related to dependencies metadata report will be excluded.
+    //
+    // Default is enabled.
+    includeDependenciesReport = true
+}
+```
+
+```kotlin
+sentry {
+    // Disables or enables the reporting of dependencies metadata for Sentry.
+    // If enabled the plugin will collect external dependencies and will take care of
+    // uploading them to Sentry as part of events. If disabled, all the logic
+    // related to dependencies metadata report will be excluded.
+    //
+    // Default is enabled.
+    includeDependenciesReport.set(true)
+}
+```
+
+## Dependencies report
+
+The plugin automatically collects external dependencies (including transitive ones) of the application and makes a report available to the runtime; the Sentry Java SDK picks the report up at runtime and sends it along with every Sentry event. The event dependencies metadata will look similar to this:
+
```

Comment:
It's actually not a runtime-, but a compile-time report, and then it gets ingested into runtime, so that the SDK can pick it up and send it with the event, so I'm not sure if this is correct

Reply:
@romtsn Ah! Got it. How about: 
```suggestion
The plugin automatically collects an application's external dependencies (including transitive ones), and generates a compile-time report. The Sentry Java SDK picks up the report at runtime and sends it along with every Sentry event. The event dependencies metadata will look something like this:

```

Reply:
Yeah, that looks good, thank you!